robosat/osm/road.py

```diff
@@ -0,0 +1,160 @@
+
+    #  highway=* to discard because these features are not vislible in satellite imagery
+    @staticmethod
+    def road_filter():
+        return {
+            "motorway",
+            "trunk",
+            "primary",
+            "secondary",
+            "tertiary",
+            "unclassified",
+            "residential",
+            "service",
+            "motorway_link",
+            "trunk_link",
+            "primary_link",
+            "secondary_link",
+            "tertiary_link"
+        }
+
+    highway_attributes = {
+        'motorway': {
+            'lanes': 4,
+            'lane_width': 3.75,
+            'left_hard_shoulder_width': 0.75,
+            'right_hard_shoulder_width': 3.0
+        },
+        'trunk': { 
+            'lanes': 3,
+            'lane_width': 3.75,
+            'left_hard_shoulder_width': 0.75,
+            'right_hard_shoulder_width': 3.0
+        },
+        'primary': { 
+            'lanes': 2,
+            'lane_width': 3.75,
+            'left_hard_shoulder_width': 0.50,
+            'right_hard_shoulder_width': 1.50
+        },
+        'secondary': { 
+            'lanes': 1,
+            'lane_width': 3.50,
+            'left_hard_shoulder_width': 0.00,
+            'right_hard_shoulder_width': 0.75
+        },
+        'tertiary': { 
+            'lanes': 1,
+            'lane_width': 3.50,
+            'left_hard_shoulder_width': 0.00,
+            'right_hard_shoulder_width': 0.75
+        },
+        'unclassified': { 
+            'lanes': 1,
+            'lane_width': 3.50,
+            'left_hard_shoulder_width': 0.00,
+            'right_hard_shoulder_width': 0.00
+        },
+        'residential': { 
+            'lanes': 1,
+            'lane_width': 3.50,
+            'left_hard_shoulder_width': 0.00,
+            'right_hard_shoulder_width': 0.75
+        },
+        'service': { 
+            'lanes': 1,
+            'lane_width': 3.00,
+            'left_hard_shoulder_width': 0.00,
+            'right_hard_shoulder_width': 0.00
+        },
+        'motorway_link': { 
+            'lanes': 2,
+            'lane_width': 3.75,
+            'left_hard_shoulder_width': 0.75,
+            'right_hard_shoulder_width': 3.00
+        },
+        'trunk_link': { 
+            'lanes': 2,
+            'lane_width': 3.75,
+            'left_hard_shoulder_width': 0.50,
+            'right_hard_shoulder_width': 1.50
+        },
+        'primary_link': { 
+            'lanes': 1,
+            'lane_width': 3.50,
+            'left_hard_shoulder_width': 0.00,
+            'right_hard_shoulder_width': 0.75
+        },
+        'secondary_link': { 
+            'lanes': 1,
+            'lane_width': 3.50,
+            'left_hard_shoulder_width': 0.00,
+            'right_hard_shoulder_width': 0.75
+        },
+        'tertiary_link': { 
+            'lanes': 1,
+            'lane_width': 3.50,
+            'left_hard_shoulder_width': 0.00,
+            'right_hard_shoulder_width': 0.00
```

Comment:
Could you run

    black . --exclude .env --check --line-length 120

for formatting? Looks like e.g. here we use `'` and below we use `"`.